wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandler.swift

```diff
@@ -266,6 +266,23 @@ final class CreateGroupConversationActionHandler: ActionHandler<CreateGroupConve
                 await self.context.perform {
                     action.succeed(with: newConversation.objectID)
                 }
+
+             } catch MLSService.MLSAddMembersError.failedToClaimKeyPackages(let failedMLSUsers) {
```

Comment:
This will report the group creation as a success but no participants will actually be added since the commit is not retried without the clients with no key packages.

I think they way to solve is to handle error in the `ConversationService` in a similar way we do it in https://github.com/wireapp/wire-ios/blob/e4b22124b181017affe6f40d1f81a740a7329e5e/wire-ios-request-strategy/Sources/Request%20Strategies/Conversation/ConversationParticipantsService.swift#L141

Reply:
ah you mean I am missing to retry for the non failing users?

Reply:
Yes, and we don't handle federation errors